raft/clock.go

```diff
@@ -16,9 +16,6 @@ const (
 
 	// DefaultReconnectTimeout is the default time to wait before reconnecting.
 	DefaultReconnectTimeout = 10 * time.Millisecond
-
-	// DefaultWaitInterval is the default time to wait log sync.
-	DefaultWaitInterval = 1 * time.Millisecond
```

Comment:
I put it in the clock originally but the wait really just needs to check over and over until the index changes. It's a pain to control via the clock. We need to find a more efficient way to do waits in the future anyway so I just removed it. It cleans up the test code.